src/pages/iou/request/step/IOURequestStepReport.tsx

```diff
@@ -31,9 +31,9 @@ function IOURequestStepReport({route, transaction}: IOURequestStepReportProps) {
     const isFromGlobalCreate = !!transaction?.isFromGlobalCreate;
     const reportOrDraftReport = getReportOrDraftReport(reportIDFromRoute);
 
-    const handleGoBackWithReportID = (id: string) => {
+    const handleGoBackWithReportID = () => {
```

Comment:
```suggestion
    const handleGoBack = () => {
```
Let's update a function name as we are no longer passing the id